sig-vendor/multipart-post/sig.rbs

```diff
@@ -0,0 +1,18 @@
+# https://github.com/socketry/multipart-post/blob/cce3634876b74e08a6c51d02db00070a82dc8cc4/lib/net/http/post/multipart.rb#L14-L28
```

Comment:
multipart-post library and community definition don't have RBS for it. This is required to pass check by Steep.